addons/ai/stringtable.xml

```diff
@@ -54,6 +59,7 @@
             <Portuguese>Precisão da Mira</Portuguese>
             <Japanese>照準精度</Japanese>
             <Korean>조준 정확도</Korean>
+            <Spanish>Precisión al apuntar</spanish>
```

Comment:
Invalid closing tag. Should be `</Spanish>` (capital S).

Reply:
Fix